content/en/blog/_posts/the-case-for-kubernetes-limits/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: "The Case for Kubernetes Resource Limits: Predictability vs. Efficiency"
+date: TODO
+slug: the-case-for-kubernetes-resource-limits-predictability-vs-efficiency
```

Comment:
I recommend:
```suggestion
slug: the-case-for-kubernetes-resource-limits
```

Most search engines etc can find the `<title>` and `<h1>` elements to learn the actual full title, and it's less to type.